src/main/java/com/theironyard/invitator/Main.java

```diff
@@ -0,0 +1,131 @@
+package com.theironyard.invitator;
+
+import spark.ModelAndView;
+import spark.Session;
+import spark.Spark;
+import spark.template.mustache.MustacheTemplateEngine;
+
+import java.util.*;
+
+/**
+ * Created by melmo on 12/22/16.
+ */
+public class Main {
+    static String SESSION_USERID = "username";
+//    static User user; // ??? Store user or get every time from db ???
+
+    public static void main(String[] args) {
+        Spark.staticFileLocation("/static");
+        Spark.init();
+
+        Spark.get("/", ((request, response) -> {
+            Map m = new HashMap();
+            Session session = request.session();
+            Integer currentUser = session.attribute(SESSION_USERID);
+
+            if (currentUser==null){
+                return new ModelAndView(m, "login.html");
```

Comment:
This and the other `return` statement are almost identical except the template.  Best practices say you should only have 1 return statement in each method/function if possible.  In this case you can just store the template as a string setting it in the if/else blocks and then return the new `ModelAndView` with the template dynamically set by the variable.